tests/utils.py

```diff
@@ -44,15 +45,15 @@ def get_test_data(sample_size=1000, embedding_size=4, sparse_feature_num=1, dens
 
     for i in range(sparse_feature_num):
         if use_group:
-            group_name = str(i%3)
+            group_name = str(i % 3)
         else:
             group_name = DEFAULT_GROUP_NAME
         dim = np.random.randint(1, 10)
         feature_columns.append(
-            SparseFeat(prefix + 'sparse_feature_' + str(i), dim, embedding_size, use_hash=hash_flag, dtype=tf.int32,group_name=group_name))
+            SparseFeat(prefix + 'sparse_feature_' + str(i), dim, embedding_size, use_hash=hash_flag, dtype=tf.int32, group_name=group_name))
 
     for i in range(dense_feature_num):
-        transform_fn = lambda x: (x - 0.0)/ 1.0
+        def transform_fn(x): return (x - 0.0) / 1.0
```

Comment:
i think lambda function is ok

Reply:
Yeah, you are right. Here `lambda` function is also ok. But when I save this python file, my `vscode` environment automatically transform the `lambda` function to `def` function. I adopt google tensorflow python style configuration in my `vscode` settings.